### PR TITLE
add viewers field in dashboard service

### PR DESCRIPTION
**Pull request by stat-kwon**
Signed-off-by: Seolmin Kwon <seolmin@megazone.com>

### Category
- [ ] New feature
- [ ] Bug fix
- [x] Improvement
- [ ] Refactor
- [ ] etc

### Description
* add viewers field in dashboard service

### CODE REVIEW

File: proto/spaceone/api/dashboard/v1/domain_dashboard.proto [hunk 1]
```diff
@@ -77,8 +77,15 @@ message DomainDashboardSettings {
 }
 
 message CreateDomainDashboardRequest {
+  enum Viewers {
+    SCOPE_NONE = 0;
```
Comment (whdalsrnt) [inline proto/spaceone/api/dashboard/v1/domain_dashboard.proto:81]: Change default value (`SCOPE_NONE` -> `VIEWERS_NONE`)

File: proto/spaceone/api/dashboard/v1/domain_dashboard.proto [hunk 2]
```diff
@@ -178,22 +185,22 @@ message DomainDashboardQuery {
...
   // is_required: false
   string user_id = 10;
   // is_required: true
   string domain_id = 11;
 }
 
 message DomainDashboardInfo {
-  enum Scope {
+  enum Viewers {
     SCOPE_NONE = 0;
```
Comment (whdalsrnt) [inline proto/spaceone/api/dashboard/v1/domain_dashboard.proto:197]: Change default value (SCOPE_NONE -> VIEWERS_NONE)

File: proto/spaceone/api/dashboard/v1/project_dashboard.proto [hunk 3]
```diff
@@ -78,12 +78,19 @@ message ProjectDashboardSettings {
 }
 
 message CreateProjectDashboardRequest {
+  enum Viewers {
+    SCOPE_NONE = 0;
```
Comment (whdalsrnt) [inline proto/spaceone/api/dashboard/v1/project_dashboard.proto:82]: Change default value (SCOPE_NONE -> VIEWERS_NONE)

File: proto/spaceone/api/dashboard/v1/project_dashboard.proto [hunk 4]
```diff
@@ -169,10 +176,10 @@ message ProjectDashboardVersionQuery {
 }
 
 message ProjectDashboardQuery {
-  enum Scope {
+  enum Viewers {
     SCOPE_NONE = 0;
```
Comment (whdalsrnt) [inline proto/spaceone/api/dashboard/v1/project_dashboard.proto:180]: Change default value (SCOPE_NONE -> VIEWERS_NONE)

**Comment by stat-kwon** [comment]
Thanks for the code review. @whdalsrnt 
After changing from SCOPE_NONE to VIEWERS_NONE, it was reflected in the PR.

**Review by whdalsrnt — APPROVED** [review]
Thanks for the quick change.

**Pull request merged**